lib/deaf_grandma.rb

```diff
@@ -21,7 +21,17 @@ def run!
   def speak(input)
 
     #Implement your code here <<<<<<<<<
-
+    if input != input.upcase
+      "SPEAK UP SONNY!"
+    elsif input == input.upcase && input != "BYE"
```

Comment:
There's something fishy about the different cases in your if statement (hint: If I am true, that means I can't be false)